src/magic.cpp

```diff
@@ -1051,6 +1051,34 @@ std::string spell::enumerate_targets() const
     return ret;
 }
 
+std::string spell::list_targeted_monster_ids() const
+{
+    if( type->targeted_monster_ids.empty() ) {
+        return "";
+    }
+    std::vector<std::string> all_valid_target_ids;
+    for( auto iter : type->targeted_monster_ids ) {
+        all_valid_target_ids.emplace_back( iter->nname() );
+    }
+    if( all_valid_target_ids.size() == 1 ) {
+        return all_valid_target_ids[0];
+    }
+
+    std::string ret;
+    // @TODO: if only we had a function to enumerate strings and concatenate them...
```

Comment:
we do have something like that. i know that for a fact. i've used it. i just can't recall what it is at this time.

Reply:
To be honest, I just copied that from your code above. 
But I'll search for that function just in case.

Reply:
i believe that too. i'll leave you to it and look around tomorrow

Reply:
There's `enumerate_as_string` in `output.h`.

Reply:
Thank you Qrox!